app/views/devise/registrations/new.html.erb

```diff
@@ -1,63 +1,69 @@
-    </div>
-  <% end %>
-
-  <div class="form-group">
-    <%= f.label :name %>
-    <%= f.text_field :name, autocomplete: 'name', class: 'form-control' %>
-  </div>
-
-   <div class="form-group">
-    <%= f.label :role %>
-    <%= f.select(:role, options_for_select([['Admin']]), {}, {class: "form-control"}) %>
-  </div>
-
-  <div class="form-group">
-    <%= f.label :department_id %>
-    <%= f.select(:department_id, options_for_select([['Development', 1],['Product', 2]]), {}, {class: "form-control"}) %>
-  </div>
-
-  <div class="form-group">
-    <%= f.label :email %>
-    <%= f.email_field :email, autocomplete: 'email', class: 'form-control' %>
-  </div>
-
-  <div class="form-group">
-    <%= f.label :password %>
-    <%= f.password_field :password, autocomplete: 'current-password', class: 'form-control' %>
-
-    <% if @minimum_password_length %>
-      <small class="form-text text-muted"><%= t('devise.shared.minimum_password_length', count: @minimum_password_length) %></small>
-    <% end %>
-  </div>
-
-  <div class="form-group">
-    <%= f.label :password_confirmation %>
-    <%= f.password_field :password_confirmation, autocomplete: 'current-password', class: 'form-control' %>
-  </div>
-
-  <div class="form-group">
-    <%= f.submit t('.sign_up'), class: 'btn btn-regular btn-primary' %>
-  </div>
-<% end %>
-
-<%= render 'devise/shared/links' %>
+<%= render 'menus/header' %>
+<div class='container'>
+	<h1><%= t('.sign_up') %></h1>
 
+	<%= nested_form_for([resource_name, resource.with_company], url: registration_path(resource_name)) do |f| %>
+	  <%= bootstrap_devise_error_messages! %>
+
+	  <%= f.fields_for :company do |company_form| %>
+	    <div class="form-group">
+	      <%= company_form.label :name, "Company Name" %>
+	      <%= company_form.text_field :name, autofocus: true, class: 'form-control', id: 'company-name' %>
+	    </div>
+	    
+	    <div class="form-group">
+	      <%= company_form.label :description, "Company Description" %>
+	      <%= company_form.text_field :description,  class: 'form-control' %>
+	    </div>
+
+	    <div class="form-group">
+	      <%= company_form.label :subdomain, "Company Subdomain" %>
+	      <%= company_form.text_field :subdomain, class: 'form-control', id: 'company-subdomain', readonly: true %>
+	    </div>
+	      <%= company_form.fields_for :setting do |setting_form| %>
+	      <%= setting_form.hidden_field :timings, :value => ('{ "monday_start_time" => "00:00 AM", "monday_end_time" => "00:00 AM", "tuesday_start_time" => "00:00 AM", "tuesday_end_time" => "00:00 AM", "wednesday_start_time" => "00:00 AM", "wednesday_end_time" => "00:00 AM", "thursday_start_time" => "00:00 AM", "thursday_end_time" => "00:00 AM", "friday_start_time" => "00:00 AM", "friday_end_time" => "00:00 AM", "saturday_start_time" => "00:00 AM", "saturday_end_time" => "00:00 AM", "sunday_start_time" => "00:00 AM", "sunday_end_time" => "00:00 AM" }') %>
```

Comment:
write a helper function to get this value

Reply:
that's just default settings, won't need that I guess as per zain